src/test/java/com/coveo/spillway/SpillwayTest.java

```diff
@@ -384,4 +394,74 @@ public void triggersAreIgnoreIfTheStorageReturnsAnIncoherentResponse() {
 
     verify(callback, never()).trigger(any(LimitDefinition.class), any());
   }
+
+  @Test
+  public void canAddCapacityLimitOverride() {
+    LimitOverride override = LimitOverrideBuilder.of(JOHN).to(A_CAPACITY).per(A_DURATION).build();
+
+    Limit<User> userLimit =
+        LimitBuilder.of("perUser", User::getName)
+            .to(A_HIGHER_CAPACITY)
+            .per(A_DURATION)
+            .withLimitOverride(override)
+            .build();
+
+    Spillway<User> spillway = inMemoryFactory.enforce("testResource", userLimit);
+    assertThat(spillway.tryCall(john, A_CAPACITY + 10)).isFalse();
+  }
+
+  @Test
+  public void canAddExpirationLimitOverride() throws InterruptedException {
+    LimitOverride override =
+        LimitOverrideBuilder.of(JOHN).to(A_CAPACITY).per(A_SHORT_DURATION).build();
+
+    Limit<User> userLimit =
+        LimitBuilder.of("perUser", User::getName)
+            .to(A_CAPACITY)
+            .per(A_DURATION)
+            .withLimitOverride(override)
+            .build();
+
+    Spillway<User> spillway = inMemoryFactory.enforce("testResource", userLimit);
+    assertThat(spillway.tryCall(john, A_CAPACITY)).isTrue();
+    assertThat(spillway.tryCall(john, 1)).isFalse();
+
+    Thread.sleep(2000);
```

Comment:
We could probably do that, that depends on how precise the Instant.now is and how much computation time is required by the various functions.
I don't like when tests fails because the test program decided to pause the test for some reason.
The best would be to mock the static call, but that would require PowerMock (which I don't really like).